### DISCUSSION TITLE
How to deploy milvus cluster in the easiest way?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [milvus-io/milvus#36331] [Q&A and General discussion]
Also if i don't want to availability, but want to increase search speed. What type of cluster should I increase:

proxy
data
query
index
coordinator

**Comment by yhmo** [comment]
Search request is executed by query nodes. You can increase CPU cores of each query node to get faster search speed.

**Comment by xiaofan-luan** [comment]
> Also if i don't want to availability, but want to increase search speed. What type of cluster should I increase:
> 
> proxy data query index coordinator

If you don't need to scale, simply deploy a standalone nodes with more resources?
